pages/api-reference/funding-sources/remove.mdx

```diff
@@ -34,7 +36,7 @@ If a funding source has an **incoming** transfer, we recommend not removing it u
 | ----------- | ------------------------- |
 | 404         | Funding source not found. |
 
-### Request and response
+### Request and response - bank account funding source
```

Comment:
Let's use "bank funding source" to stay consistent with other mentions of this term.